quadrat/block-template-parts/header.html

```diff
@@ -1,8 +1,8 @@
-<!-- wp:group {"tagName":"header","className":"site-header"} -->
-<header class="wp-block-group site-header">
+<!-- wp:group {"tagName":"header","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"170px","left":"0px"}}},"className":"site-header"} -->
```

Comment:
The header in this PR appears for me without correct padding:

![Screen Shot 2021-07-09 at 11 10 13 AM](https://user-images.githubusercontent.com/5375500/125099675-457dd700-e0a6-11eb-81f4-008063832bd2.png)

I think the top, right, and left padding values should be removed, and it works as expected.

Reply:
Ah I didn't spot this but I can see it too, will sort it!

Reply:
Fixed this now and pushed it, thanks @jffng!